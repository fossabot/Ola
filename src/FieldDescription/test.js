import React from 'react'
import FieldDescription from './'
import renderer from 'react-test-renderer'

it('Default FieldDescription', () => {
  const tree = renderer
    .create(<FieldDescription>Description</FieldDescription>)
    .toJSON()
  expect(tree).toMatchSnapshot()
})

it('FieldDescription with error', () => {
  const tree = renderer
    .create(<FieldDescription error={true}>Description</FieldDescription>)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
