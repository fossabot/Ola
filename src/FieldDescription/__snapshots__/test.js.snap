// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Default FieldDescription 1`] = `
<p
  className="ola_field-description"
>
  Description
</p>
`;

exports[`FieldDescription with error 1`] = `
<p
  className="ola_field-error"
>
  Description
</p>
`;
